### ISSUE TITLE
`delete` statement dropped in the output

### ISSUE THREAD

**Issue by pygy** [rollup/rollup#352]
```JS
//main.js
export default function foo(o) {}

foo.bar = 8;

delete foo.bar;
```

`rollup -i main.js > bundle.js`

```JS
//bundle.js
function foo(o) {}

foo.bar = 8;
// note the conspicuous absence of `delete foo.bar;`.
export default foo;
```
Is this expected?

The test case is available in the [`drops-delete` branch of my rollup playground](https://github.com/pygy/trollup/tree/drops-delete).

**Comment by Victorystick** [comment]
@pygy Nice catch!

**Comment by pygy** [comment]
I've simplified the test case a bit.

**Comment by Rich-Harris** [comment]
[Fixed in 0.21.3.](http://rollupjs.org/#%7B%22options%22%3A%7B%22format%22%3A%22cjs%22%2C%22moduleName%22%3A%22myBundle%22%2C%22globals%22%3A%7B%7D%7D%2C%22modules%22%3A%5B%7B%22name%22%3A%22main.js%22%2C%22code%22%3A%22export%20var%20foo%20%3D%20%7B%20bar%3A%202%20%7D%3B%5Cndelete%20foo.bar%3B%22%7D%5D%7D) Thanks!

**Comment by pygy** [comment]
Great, thanks :-)
